vmdb/app/models/mixins/has_many_orchestration_stack_mixin.rb

```diff
@@ -7,8 +7,8 @@ module HasManyOrchestrationStackMixin
              :dependent   => :destroy
 
     has_many :direct_orchestration_stacks,
+             -> { where(:orchestration_stacks => {:ancestry => nil}) },
```

Comment:
Actually I also tested `->  { where("ancestry is NULL") }` and it worked too. Which syntax do we prefer?